src/webapp/components/import-export/ImportTable.tsx

```diff
@@ -597,14 +630,23 @@ const useValidations = (
             return {
                 validation: (value: string) => {
                     if (!value) return i18n.t("Please provide a username");
+                    if (/^[._@-]|[._@-]$/.test(value)) {
```

Comment:
These new validations are business rules validations and should be realized in the user domain entity